modules/masterbar/admin-color-schemes/colors/_core-overrides.scss

```diff
@@ -0,0 +1,48 @@
+/* Masterbar */
+#wpadminbar {
+	-webkit-box-shadow: inset 0 -1px 0 $menu-submenu-background;
+	-moz-box-shadow: inset 0 -1px 0 $menu-submenu-background;
+	box-shadow: inset 0 -1px 0 $menu-submenu-background;
+}
+
+#wpadminbar .ab-top-menu > #wp-admin-bar-blog > .ab-item {
+	background: $menu-submenu-background;
+	color: $text-color !important;
+}
+
+#wpadminbar .ab-top-menu > #wp-admin-bar-newdash > .ab-item {
+	color: $text-color !important;
+}
+
+#wpadminbar:not(.mobile) .ab-top-menu > li:hover > .ab-item,
+#wpadminbar #wp-admin-bar-notes.hover > .ab-item {
+	background: $menu-submenu-background;
+}
+
+/**
+ * Site Card
+ */
+ #adminmenu li.toplevel_page_site-card {
```

Comment:
Looks like there's unnecessary whitespace at the beginning of the line here

Reply:
Removed 40d2b27